src/main/java/org/example/final02/model/entity/Authors.java

```diff
@@ -0,0 +1,23 @@
+package org.example.final02.model.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Authors {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long idAuthor;
+
+    private String firstNameAuthor;
+    private String lastNameAuthor;
```

Comment:
There is missing mapping

```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long idAuthor;

    private String firstNameAuthor;
    private String lastNameAuthor;
    
     @OneToMany(mappedBy = "author")
    private List<Book> books;
```